guides.dataverse.org.xml

```diff
@@ -0,0 +1,76 @@
+
+# 2016-03-08 PDF Y U NO BUILD?!?
+#(cd $WORKSPACE/doc/sphinx-guides; make clean; scl enable python27 &quot;make html&quot;;  scl enable python27 &quot;make epub&quot;; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@guides.dataverse.org:/tmp)
+
+
+#(cd $WORKSPACE/doc/sphinx-guides; make clean; scl enable python27 &quot;make html&quot;; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@docs.dataverse.org:/tmp)
+
+# New and improved command line to eliminate epub unknown mime-type warnings, see github #3391
+#(cd $WORKSPACE/doc/sphinx-guides; make clean; scl enable python27 &quot;make html SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; scl enable python27 &quot;make latexpdf SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; cp build/latex/Dataverse.pdf build/html; scl enable python27 &quot;make epub SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@docs.dataverse.org:/tmp)
+
+# docs.dataverse.org is no longer in DNS :( -- pdurbin 2018-03-13
+#(cd $WORKSPACE/doc/sphinx-guides; make clean; scl enable python27 &quot;make html SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; scl enable python27 &quot;make latexpdf SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; cp build/latex/Dataverse.pdf build/html; scl enable python27 &quot;make epub SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@docs.dataverse.org:/tmp)
+
+#ssh -l user docs.dataverse.org &quot;(mkdir -p /var/www/docs.dataverse.org/en/$VERSION &amp;&amp; cd /var/www/docs.dataverse.org/en/$VERSION &amp;&amp; tar xvzf /tmp/$TARBALL)&quot;
+
+#(cd $WORKSPACE/doc/sphinx-guides; make clean; scl enable python27 &quot;make html SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; scl enable python27 &quot;make epub SPHINXBUILD=/nfs/tools/lib/python/2.7/current/bin/sphinx-build&quot;; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@guides.dataverse.org:/tmp)
+
+#original build command with PDF
+#(cd $WORKSPACE/doc/sphinx-guides; make clean; make html; make latexpdf; cp build/latex/Dataverse.pdf build/html; make epub; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@guides.dataverse.org:/tmp)
+
+# build without latexPDF which doesn&apos;t compile
+(cd $WORKSPACE/doc/sphinx-guides; make clean; make html; make epub; cp build/epub/Dataverse.epub build/html; cd build/html; tar cvzf /tmp/$TARBALL .; scp /tmp/$TARBALL user@guides.dataverse.org:/tmp)
```

Comment:
Is this scp to guides.dataverse.org real? Somebody went and created a user called "user" on that server?